gz_rviz/package.xml

```diff
@@ -22,9 +21,15 @@
   <depend>visualization_msgs</depend>
 
   <!-- Fortress (default) -->
+  <depend condition="$IGNITION_VERSION == 'fortress'">ignition-common4</depend>
   <depend condition="$IGNITION_VERSION == 'fortress'">ignition-gui6</depend>
+  <depend condition="$IGNITION_VERSION == 'fortress'">ignition-math6</depend>
```

Comment:
Do these need to be `GZ_VERSION` ?

Reply:
https://github.com/gazebosim/gz-rviz/pull/85/commits/dec5acb5b14ca3e866e706e0465d1ed2902b2e90